validate_clerk_input.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Aug 19 18:26:19 2021
+
+@author: vjham
+"""
+
+from product_list import products
+
+
+def validate_clerk_input(clerk_input):
+    
+    list_of_valid_product_ids = [str(product['id']) for product in products]
+    
+    if clerk_input in list_of_valid_product_ids:
+        return clerk_input
+    elif clerk_input.upper() == 'DONE':
```

Comment:
So - I think this is a great change - comparing clerk_input.upper() to 'DONE'. BUT. I don't think it belongs in this PR. I would again make a separate GitHub issue for this one. And leave the current logic as is. (This goes back to my whole approach of always preparing for the worst-case scenario: If we decide to rollback the input validation changes, we wouldn't want this change to get rolled back with it, because this is a good change.)

Reply:
I saw it as a necessary part of input validation. I think we should talk about this today, so you can help me understand what language you would put in the separate issue.

Reply:
Actually - just thinking - we don't need to make a separate GitHub issue for this - we kind of already have one: Issue #20